pipelines/cogito.yml

```diff
@@ -57,9 +57,10 @@ resources:
 - name: repo.git
   type: git
   source:
-    uri: git@github.com:((github-owner))/((repo-name)).git
+    uri: https://github.com/((github-owner))/((repo-name)).git
+    #uri: git@github.com:((github-owner))/((repo-name)).git
```

Comment:
leftover

Reply:
Then, shall we remove the private_key above ?

Reply:
I understand that it can be confusing at first sight.
I left the commented-out line and the SSH private_key there on-purpose: it is up to the owner of the pipeline to adapt according to wether the target repo is public or private.
Your comment makes me think I can make it more evident; I will push a change.

Reply:
Changed.